.github/workflows/docs.yml

```
@@ -0,0 +1,17 @@
+name: Docs build
+
+on: [push, pull_request]
+
+jobs:
+  docs build:
```

```suggestion
  docs_build:
```
I think this action is not running due to this space which is not allowed in the job name https://github.com/vitessce/vitessce-python/actions/runs/2155324608